src/ImageGallery.js

```diff
@@ -239,10 +239,12 @@ class ImageGallery extends React.Component {
     const { onThumbnailClick } = this.props;
     // blur element to remove outline cause by focus
     event.target.parentNode.parentNode.blur();
+   if(this.state.currentIndex!==index){
     this.slideToIndex(index, event);
     if (onThumbnailClick) {
```

Comment:
lets move this conditional out of this logic since we still want to track thumbnail click events.